docs/ContribOperators.md

```diff
@@ -1185,6 +1186,58 @@ This version of the operator has been available since version 1 of the 'com.micr
 </dl>
 
 
+### <a name="com.microsoft.GridSample"></a><a name="com.microsoft.gridsample">**com.microsoft.GridSample**</a>
+
+  Given an `input` and a flow-field `grid`, computes the `output` using `input` values and pixel locations from `grid`.
+        Currently, only spatial (4-D) inputs are supported. For `input` with shape (N, C, H, W) and `grid` with shape (N, H_out, W_out, 2),
+        the `output` will have shape (N, C, H_out, W_out).
+        For each output location `output[n, :, h, w]`, the size-2 vector `grid[n, h, w]` specifies `input` pixel locations `x` and `y`,
```

Comment:
in PyTorch grid_sample, the input/output shape could be different: `input[n, c, h_in, w_in]`, `output[n, c, h_out, w_out]`. 

https://pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html#torch.nn.functional.grid_sample

Is it intended to design contrib GridSample having same shape for input and output?

Reply:
yes, it's the same with `output` and `grid`. The description may be not very clear. I will amend this description.